Client/Frontend/Browser/RecentlyClosedTabs.swift

```diff
@@ -0,0 +1,97 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
If this file is in the Storage target, this file should be in `Storage` and not `Client/Frontend/Browser`.